.api_violations.report

```diff
@@ -1,6 +0,0 @@
-API rule violation: names_match,github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh,NetworkService,NetworkServiceName
```

Comment:
@edwarnicke Why does this file keep getting deleted with each of your PRs? I believe we need this file, as I saw @sbezverk re-add it the last time you deleted it.

Reply:
@mestery The magic updating scripts do it... I don't know *why* they do it, but if I don't run:

```GOPATH=${GOPATH} ./scripts/update-codegen.sh && ./scripts/add-openapi-bits.sh```

When I update common.proto, things break.  If I do run it, it empties out the .api_violations.report file :(